### PR TITLE
Add WriteStream

### PR DESCRIPTION
**Pull request by mrmacete**

### CODE REVIEW

**Comment by oleavr** [comment]
Awesome! 🤘

**Comment by oleavr** [comment]
+ frida-fs@1.1.0

File: index.js [hunk 1]
```diff
@@ -48,8 +48,53 @@ class ReadStream extends stream.Readable {
...
+    this._writeRequest = null;
+
+    const pathStr = Memory.allocUtf8String(path);
+    /*
+     * flags = O_WRONLY | O_CREAT (= 0x201)
+     * mode  = 0644               (= 0x1a4)
+     */
+    const fd = open(pathStr, 0x201, 0x1a4);
+    if (fd === -1) {
+      this.emit('error', new Error('Unable to open file'));
```
Comment (dweinstein) [inline index.js]: this error emit should probably happen in a process.nextTick as the error might be emitted before someone is able to attach

Comment (dweinstein) [inline index.js]: or process.setImmediate I guess http://peterlyons.com/problog/2014/03/settimeout-and-friends

Comment (oleavr) [inline index.js]: Nice catch! It's currently impossible to receive the `error` event.

Comment (oleavr) [inline index.js]: Yeah that sounds like the best option. I'll implement `setImmediate()` in Frida shortly.

**Pull request merged**